core/src/main/java/nl/nn/adapterframework/http/cxf/SOAPProviderBase.java

```diff
@@ -89,8 +89,8 @@ public SOAPMessage invoke(SOAPMessage request) {
 		Message response;
 		try (PipeLineSession pipelineSession = new PipeLineSession()) {
 			String messageId = UUIDUtil.createSimpleUUID();
-			PipeLineSession.updateListenerParameters(pipelineSession, messageId, messageId, new Date(), null);
-			log.debug((messageId)+"received message");
+			PipeLineSession.updateListenerParameters(pipelineSession, messageId, messageId, Instant.now(), null);
```

Comment:
misschien handiger hier null,null terug te geven ipv zelf instant.now

Reply:
Maar het was `new Date()` en dat is hetzelfde als `Instant.now()`. Dus nu functioneel niets gewijzigd.

Reply:
Klopt, en als je null null teruggeeft veranderd er functioneel ook niets, maar dan schoon je de code wel iets wat op.

Reply:
Ik heb #5924 aangemaakt.